primitives/core/src/assertion.rs

```diff
@@ -213,6 +217,8 @@ impl Assertion {
 			Self::Achainable(a) => vec![a.chain()],
 			// Oneblock Assertion
 			Self::Oneblock(..) => vec![Web3Network::Polkadot, Web3Network::Kusama],
+			// SPACEID Assertions
+			Self::SPACEIDBnbDomainHolding => vec![Web3Network::Bsc],
```

Comment:
Is there any downside to remove the SPACEID, so we could make this as generic as possible?
i.e. there is other partner wanted bnbdomainholding, we can reuse this.

Reply:
No, thanks. I will remove it.